src/applications/benefit-eligibility-questionnaire/containers/ConfirmationPage.jsx

```diff
@@ -124,6 +109,22 @@ export class ConfirmationPage extends React.Component {
     this.setState({ sortValue: sortStrings[key] });
   };
 
+  sortBenefitObj(benefitObj, sortKey) {
```

Comment:
It looks like the new code at line 112 introduces a new linting warning. Can we please clean these up before we merge this?

Reply:
Ok, this is resolved now.